src/containers/Main/SiteLayout.tsx

```diff
@@ -51,7 +51,9 @@ const SiteLayout: React.FC = () => {
                 <Home />
               </Route>
               <Route exact path="/login">
-                <Login />
+                <ApplicationProvider>
+                  <Login />
+                </ApplicationProvider>
```

Comment:
Why the need for adding the ApplicationProvider here?

Reply:
So it can access the User context to write the user info to context. But if we're not going to do that here, then we can take this provider away.